lib/Model.js

```diff
@@ -206,6 +206,45 @@ Model.compile = function compile (name, schema, options, base) {
     return table.buildTableReq(table.name, table.schema);
   };
 
+  function createTransactionFunction(func, val, optionsIndex, model) {
+    return async function () {
+      let args = [...arguments];
+      if (typeof args[args.length - 1] === "function") {
+        // Callback function passed in which is not allowed, removing that from arguments list
```

Comment:
Would probably be a good idea to warn about that - might lead to unexpected results, if callback is being passed in, and never called.

Reply:
@jutaz Think we should do this as a `console.warning` or `debug`?

Reply:
I'd do `console.warning` most of people (myself included) don't run local applications with debug turned on, unless there's a specific issue I'm debugging

Reply:
Just addressed this in lasted commit.